[bugfix] worker is not respected when loadbalance:false (#51041)
elastic/beats: docs/reference/metricbeat/elasticsearch-output.md

### `loadbalance` [_loadbalance]

When `loadbalance: true` is set, Metricbeat connects to all configured hosts and sends data through all connections in parallel. If a connection fails, data is sent to the remaining hosts until it can be reestablished. Data will still be sent as long as Metricbeat can connect to at least one of its configured hosts.

> When `loadbalance: false` is set, Metricbeat sends data to a single host at a time. Metricbeat connects to the first configured host and sends all data to that host until the connection fails, then fails over to the next configured host in order. Data will still be sent as long as Metricbeat can connect to at least one of its configured hosts.

Reviewer: ```suggestion
Set `loadbalance: false` to send data to one host at a time. Metricbeat connects to the first configured host and sends all data to that host. If the connection fails, Metricbeat fails over to the next configured host. As long as Metricbeat can connect to at least one configured host, it continues to send data.
```
